facts/cologne/cologne

```diff
@@ -0,0 +1 @@
+A fragrance that makes you smell like German bratwurst.
```

Comment:
Question. What level of testing have you performed on this change? 
I'd like to see some UTs for this as well.